snow/consensus/snowman/topological.go

```diff
@@ -677,3 +677,14 @@ func (ts *Topological) rejectTransitively(ctx context.Context, rejected []ids.ID
 	}
 	return nil
 }
+
+func (ts *Topological) GetParent(id ids.ID) ids.ID {
+	if block, ok := ts.blocks[id]; ok {
+		if block.blk == nil {
+			return ids.Empty
+		}
+		return block.blk.Parent()
```

Comment:
> Can we skip block.blk == nil? We should have the invariant that block.blk is never nil since we only assign to it in one place and always with a value that we've already called Parent() on


We have the invariant, however, what if we won't have it? I think it's better to be safe than sorry and just keep this check for safety to avoid crashing the node.

>  Could we switch from using ids.Empty to signify that we don't have the block to returning an extra boolean?

Yes I switched.